src/nix/flake.md

```diff
@@ -282,7 +282,10 @@ derivation):
 }
 ```
 
-The following attributes are supported in `flake.nix`:
+The following attributes are supported in `flake.nix`, but all attributes other than `outputs`
```

Comment:
This does not read very well ("The following attributes..." followed by two sentences that are *not* the list of attributes).  Better to phrase it like "Flake attribute must be written using a limited subset bla bla... The following attributes are supported:".